rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -1066,6 +1066,12 @@ public boolean hasBtcBlockCoinbaseTransactionInformation(Object[] args) {
         return bridgeSupport.hasBtcBlockCoinbaseTransactionInformation(blockHash);
     }
 
+    public long getActiveFederationCreationBlockHeight(Object[] args) {
+        logger.trace("getActiveFederationCreationBlockHeight");
```

Comment:
Maybe, we could add the result into the log message

Reply:
well, we could, but then it won't be consistent with other trace messages in this class.